cookbooks/bcpc/libraries/utils.rb

```diff
@@ -98,11 +98,18 @@ def get_config(key, item=node.chef_environment, bag="configs")
       return value
     end
   end
+
+def delete_config(key)
+  init_config if $dbi.nil?
+  $dbi.delete(key) if $dbi.has_key?(key)
+  $dbi.save
+end
+
 end
```

Comment:
This ends `get_config()` up above and masks `delete_config()` even after adding `bcpc` as a dependency for `bach_krb5`.